src/paratii.js

```diff
@@ -80,6 +85,87 @@ class Paratii extends ParatiiCore {
   getAccount () {
     return this.eth.getAccount()
   }
+  /**
+   * Pings the provider to which the web3 is configured to connect to (see the set up in paratii.eth.js constructor)
+   * @return {Promise} that resolves in a boolean
+   */
+  async pingEth () {
```

Comment:
to be consistent, let's use `check` (just as below) here instead of `ping`

Reply:
Done in: https://github.com/Paratii-Video/paratii-js/pull/266/commits/114c3ada630021eb9c485f2b2fc010df317aa966